conf.py

```diff
@@ -44,7 +44,7 @@
 
 # General information about the project.
 project = u'Singularity container'
-copyright = u'2018, Singularity'
+copyright = u'2017-2019, Singularity'
```

Comment:
Ummm, I think it should be Sylabs instead of Singularity:

```suggestion
copyright = u'2017-2019, Sylabs inc'
```

<br>

PING @gmkurtzer, about this...

Reply:
Current now: ![image](https://user-images.githubusercontent.com/2277700/51646846-98b54b00-1f7a-11e9-96bc-f585ace43360.png)